app/src/main/res/values/strings.xml

```diff
@@ -214,34 +214,34 @@
     <string name="autofill_service_name">KeePass DX Autofill Service</string>
     <string name="autofill_sign_in_prompt">Sign in with KeePass DX</string>
     <string name="set_autofill_service_title">Set default Autofill service</string>
-    <string name="set_autofill_service_summary">Enable the service to easily fill out forms from other apps</string>
+    <string name="set_autofill_service_summary">Enable the service to quickly fill out forms from other apps.</string>
```

Comment:
The idea of ending descriptions full stop is because it is a sentence. Unless it is done, one ends up having to do it anyway for descriptions with two sentences. Unless going full GNOME and doing only period after the first sentence.

Reply:
I agree, for two sentences we are obliged. But for a parameter summary, unless you have no choice, there is only one sentence or a few words. There is no point because it is heavy and it is to stay consistent with the system applications.